cnitool/constants_windows.go

```diff
@@ -0,0 +1,22 @@
+// Copyright 2023 CNI authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+const (
+	// NOTE: this is the most reasonable default as most CNI setups on Windows
+	// will have been made using the helper script in the containerd repo.
+	// https://github.com/containerd/containerd/blob/main/script/setup/install-cni-windows
+	DefaultNetDir = "C:\\Program Files\\containerd\\cni\\conf"
```

Comment:
@aznashwan this seems containerd specific, but CNI is used by more than just containerd. If each CRI implementation has its own default directory, should this be a list of default dirs that get tried in sequence?

Reply:
Sorry for the huge delay, I must've somehow missed this notification when it happened.

> seems containerd specific

Correct.

> CNI is used by more than just containerd

Correct, but generally not really Windows (at least when I was involved in CNI porting efforts on Windows a whole year ago).

Back then, the vast majority of k8s/docker setups on Windows directly installed containerd from their build scripts in the main `containerd/containerd` repo, so it seemed like the most reasonable choice...

I'm not sure whether there is a "standardised way" to install k8s on Windows nowadays, but I'm willing to bet it uses containerd with this default.

Plus at the end of the day this path is configurable via an env var, so the `C:\Program Files\containerd\cni\conf` default isn't preventing anyone from using `cnitool`, and it's certainly a hell of a lot saner of a default on Windows than `/etc/cni/net.d` 🤣